RestVirtualDataSourceDataProviderWorker.ts

```diff
@@ -625,6 +633,18 @@ export class RestVirtualDataSourceDataProviderWorker extends AsyncVirtualDataSou
             
 			if (items && items.length > 0) {
                 let schema = this.resolveSchemaFromItems(items);
+				if (this._schemaIncludedProperties != null) {
+					let propertyNames: any[] = [];
+                	let propertyTypes: any[] = [];
+					for (let i = 0; i < schema.propertyNames.length; i++) {
+						if (!this._schemaIncludedProperties.has(schema.propertyNames[i])) {
+							continue;
+						}
+						propertyNames.push(schema.propertyNames[i]);
+						propertyTypes.push(schema.propertyTypes[i]);
+					}
+					schema = new DefaultDataSourceSchema(propertyNames, propertyTypes, schema.primaryKey, schema.propertyDataIntents);
```

Comment:
I know the PR has already been merged but isn't the constructor here missing a 5th argument subSchemas?